src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -0,0 +1,183 @@
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=135263", RemotingCapability = RemotingCapability.SupportedByCommand)]
+    public sealed class StopComputerCommand : PSCmdlet, IDisposable
+    {
+        #region Private Members
+
+        private Process _process = null;
+
+        #endregion
+
+        #region "Parameters"
+
+        /// <summary>
+        /// The authentication options for CIM_WSMan connection.
+        /// </summary>
+        [Parameter]
+        [ValidateSet(
+            "Default",
+            "Basic",
+            "Negotiate", // can be used with and without credential (without -> PSRP mapped to NegotiateWithImplicitCredential)
+            "CredSSP",
+            "Digest",
+            "Kerberos")] // can be used with explicit or implicit credential
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
+        public string WsmanAuthentication { get; set; } = "Default";
+
+        /// <summary>
+        /// The following is the definition of the input parameter "ComputerName".
+        /// Value of the address requested. The form of the value can be either the
+        /// computer name ("wxyz1234"), IPv4 address ("192.168.177.124"), or IPv6
+        /// address ("2010:836B:4179::836B:4179").
+        /// </summary>
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = true)]
+        [ValidateNotNullOrEmpty]
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        [Alias("CN", "__SERVER", "Server", "IPAddress")]
+        public string[] ComputerName { get; set; } = new string[] { "." };
+
+        /// <summary>
+        /// The following is the definition of the input parameter "Credential".
+        /// Specifies a user account that has permission to perform this action. Type a
+        /// user-name, such as "User01" or "Domain01\User01", or enter a PSCredential
+        /// object, such as one from the Get-Credential cmdlet.
+        /// </summary>
+        [Parameter(Position = 1)]
+        [ValidateNotNullOrEmpty]
+        [Credential]
+        public PSCredential Credential { get; set; }
+
+        /// <summary>
+        /// Force the operation to take place if possible.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter Force { get; set; } = false;
+
+        #endregion "parameters"
+
+        #region "IDisposable Members"
+
+        /// <summary>
+        /// Dispose Method.
+        /// </summary>
+        public void Dispose()
+        {
+            try
+            {
+                _process.Dispose();
+            }
+            catch (ObjectDisposedException) { }
+        }
+
+        #endregion "IDisposable Members"
+
+        #region "Overrides"
+
+        /// <summary>
+        /// BeginProcessing.
+        /// </summary>
+        protected override void BeginProcessing()
+        {
+            if (ComputerName.Length > 1) {
```

Comment:
No longer applicable once the `-ComputerName` param is removed

Reply:
Removed (and removed the error message I added too)

Reply:
Also added a TODO to consider stopping remote machines (via ssh)